### PR TITLE
Update ReadMe with additional information

### PR DESCRIPTION
**Pull request by srmibextesting**

### CODE REVIEW

**Review by srmibextesting — COMMENTED** [review]
Sample Comment

File: README.md [hunk 1]
```diff
@@ -6,4 +6,11 @@ Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots
 
 The standard chunk of Lorem Ipsum used since the 1500s is reproduced below for those interested. Sections 1.10.32 and 1.10.33 from "de Finibus Bonorum et Malorum" by Cicero are also reproduced in their exact original form, accompanied by English versions from the 1914 translation by H. Rackham.
 
-Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
```
Comment (srmibextesting) [inline README.md:9]: Another Comment

**Review by swissbusybee — COMMENTED** [review]
Leaving Comment

File: README.md [hunk 2]
```diff
@@ -6,4 +6,11 @@ Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots
 
 The standard chunk of Lorem Ipsum used since the 1500s is reproduced below for those interested. Sections 1.10.32 and 1.10.33 from "de Finibus Bonorum et Malorum" by Cicero are also reproduced in their exact original form, accompanied by English versions from the 1914 translation by H. Rackham.
```
Comment (swissbusybee) [inline README.md:7]: Another Comment

File: README.md [hunk 3]
```diff
@@ -6,4 +6,11 @@ Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots
...
+Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
+It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using 'Content here, content here', making it look like readable English. Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model text, and a search for 'lorem ipsum' will uncover many web sites still in their infancy. Various versions have evolved over the years, sometimes by accident, sometimes on purpose (injected humour and the like).
```
Comment (swissbusybee) [inline README.md:10]: Yet another comment

**Review by swissbusybee — APPROVED** [review]
Approving

**Comment by roman-mibex** [comment]
A later comment

**Comment by swissbusybee** [comment]
Comment Again

**Pull request merged**